packages/workspace/src/command-line/dep-graph.ts

```diff
@@ -161,7 +163,7 @@ export function generateGraph(
     return a.name.localeCompare(b.name);
   });
 
-  if (args.focus !== undefined) {
+  if (args.focus) {
```

Comment:
@philipjfulcher can you check if these `!== undefined` are needed here?

Reply:
Confirmed they're not needed. :+1: